tests/e2e/specs/backend/catalog-sorting.test.js

```diff
@@ -50,17 +44,27 @@ describe( `${ block.name } Block`, () => {
 		} );
 
 		it( 'can be inserted in FSE area', async () => {
-			await insertBlock( block.name );
```

Comment:
Do you have any idea why `insertBlock` doesn't work here?

Reply:
Tbh I'm not sure, I've tried so many things and this is the only thing that worked 😞

Reply:
> Tbh I'm not sure, I've tried so many things and this is the only thing that worked 😞

Yeah, it is very strange because we use this function everywhere. Do you think we should add a comment, just in case, for posterity?